tuf-notary/tuf-repository.go

```diff
@@ -0,0 +1,18 @@
+package tufnotary
+
+import (
+	"github.com/theupdateframework/go-tuf"
+	util "github.com/theupdateframework/go-tuf/util"
+)
+
+func Init(repository string) error {
+	var p util.PassphraseFunc
+	//TODO: get passphrase
```

Comment:
So key files are not encrypted by default, right?

Reply:
Yes, I plan to add encryption here either in an iteration of this pr or a future pr.